### DISCUSSION TITLE
How to use react-window or any other infinite scroll library with Autocomplete and other select components?

### DISCUSSION THREAD

**Discussion by orichards** [mantinedev/mantine#2071] [Q&A] [answered]
Hello!

Thank you for the wonderful components library. It's such a gem. Documentation is perfect but I still have question. How could I implement infinite-scroll within Autocomplete / MultiSelect using react-window or some other library? The goal is to open SelectList (when query is empty) and allow user to scroll through all possible choices loaded in chunks from server.

Thanks a lot.

**Comment by rtivital** [comment, marked as answer]
hi @orichards, thanks for kind words! None of mantine components support virtual scrolling, usually you can use searchable Select with `limit` prop to solve performance issue.

**Reply by orichards** [reply]
Thank you!
